.github/workflows/tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Tests
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version: ['3.8', '3.9', '3.10', 'pypy-3.7-nightly']
+
+    steps:
+
+    - name: Setup Julia
+      uses: julia-actions/setup-julia@v1
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - if: startsWith(matrix.python-version, 'pypy') != true
+      name: Install HPy (only for CPython)
+      run: |
+        # git clone -b release/0.0.3 --single-branch https://github.com/hpyproject/hpy
+        # cd hpy
+        # pip install .
+        pip install hpy>=0.9.0rc1
+
+    - name: Install dependencies
+      run: |
+        pip install numpy cython pytest transonic pythran
```

Comment:
Pythran does not work with PyPy.

Reply:
Ahh, maybe it does somewhat. At least it passes some tests https://github.com/pypy/binary-testing/actions/runs/14425922168

Reply:
At least on Linux, Pythran and PyPy are compatible. It works well at home and I'm confident that I'll be able to fix the CI.